meta-iot2000-example/recipes-support/swupdate/swupdate/defconfig

```diff
@@ -0,0 +1,75 @@
+#
+# Automatically generated file; DO NOT EDIT.
```

Comment:
Yet another `defconfig`? What's this for? There's been in the last commit as well...

Reply:
The 2nd commit adds at least the LUA handler. There are some options that can be thrown out. But in principle that's exactly like you suggested: first a small default config and then additions to the config. Isn't it?